lib/bracket-matcher-view.coffee

```diff
@@ -124,11 +124,18 @@ class BracketMatcherView
       else
         @editor.backspace()
 
+  positionIsInComment: (position) ->
+    !!@editor.scopeDescriptorForBufferPosition(position).
```

Comment:
This should use something like: https://github.com/atom/bracket-matcher/blob/28accc896c792f189aea29c58b79f4c321356b9b/lib/tag-finder.coffee#L20-L22 instead for consistency